org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/CompilationUnitResolver.java

```diff
@@ -1004,9 +1004,12 @@ private void resolve(
 						ast.setDefaultNodeFlag(0);
 						ast.setOriginalModificationCount(ast.modificationCount());
 
-						// pass it to requestor
-						astRequestor.acceptAST(source, compilationUnit);
-
+						try {
+							// pass it to requestor
+							astRequestor.acceptAST(source, compilationUnit);
+						} catch (RuntimeException e) {
+							throw new RuntimeException("Error on " + source.getPath(), e); //$NON-NLS-1$
```

Comment:
Shouldn't this be a translated string since it will appear with translated dialog text in your example?

Reply:
> Shouldn't this be a translated string since it will appear with translated dialog text in your example?

Typically we don't have translated Strings in Exceptions. We won't like to get Issues reported with exceptions in arbitrary user language. 
Priority is that those Exceptions just do not occur.

Reply:
Ok